examples/real-world/webpack.config.js

```diff
@@ -14,7 +14,12 @@ module.exports = {
   },
   plugins: [
     new webpack.optimize.OccurenceOrderPlugin(),
-    new webpack.HotModuleReplacementPlugin()
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify(process.env.NODE_ENV)
```

Comment:
That won't work either, it'll never be in production. We don't even have a script to run this in production, see the package.json: https://github.com/JeremyBernier/redux/blob/patch-1/examples/real-world/package.json